tests/verifier/lib/UDS.pm

```diff
@@ -0,0 +1,69 @@
+##
+# @file UDS.pm
+#
+# @brief Test Suite Unix Domain Socket.
+#
+#
+
+## @class
+# Container for common data.
+package UDS;
+
+use strict;
+use warnings;
+
+
+# Own modules
+use TE::Common;
+use TE::Funclet;
+use TE::Utility;
+
+
+our $test_suite_uds = 
+    [
+{
+            name => 'tc1',
+            note => '#1 - ping-pong w/o arguments',
+            pre_proc => \&te_def_pre_proc,
+            server_proc => \&te_def_server_proc,
+            server_arg => 'sr -i TARGET() ',
+            client_proc => \&te_def_client_proc,
+            client_arg => 'pp -i TARGET() ',
+            result_proc => \&te_def_result_proc,
+            result_arg => {
+                server => {
+                    success => ['Test end', 'interrupted by', 'exit'],
```

Comment:
please consider adding unix specific output matching values

Reply:
for which part? 
the output is the same as any other protocl, it's sockperf output for success

Reply:
> for which part? the output is the same as any other protocl, it's sockperf output for success

You add test specific for unix socket. Sockperf prints output related unix socket. So please consider checking this output in test adding as match criteria.